tests/integration/explain/debug/top_with_max_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package test_explain_debug
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+	explainUtils "github.com/sourcenetwork/defradb/tests/integration/explain"
+)
+
+var topLevelMaxPattern = dataMap{
+	"explain": dataMap{
+		"operationNode": []dataMap{
+			{
+				"topLevelNode": []dataMap{
+					{
+						"selectTopNode": dataMap{
+							"selectNode": dataMap{
+								"scanNode": dataMap{},
+							},
+						},
+					},
+					{
+						"maxNode": dataMap{},
+					},
```

Comment:
question: This is not under the `selectTopNode`? Is a sibling to `selectTopNode` might be worth documenting that at planner level if not already (maybe I missed it).

Reply:
In this test case it's used from the top level.